en/migrate-tidb-to-kubernetes.md

```diff
@@ -0,0 +1,119 @@
+
+    ```yaml
+    spec
+      ...
+      PDAddresses:
+      - http://pd1_addr:port
+      - http://pd2_addr:port
+      - http://pd3_addr:port
+    ```
+
+3. Confirm that the source cluster and the target cluster compose of a new cluster that runs normally:
+
+    - Get the the number and state of stores in the new cluster:
+
+        {{< copyable "shell-regular" >}}
+        
+        ```shell
+        # Get the number of stores
+        pd-ctl -u http://<address>:<port> store | jq '.count'
+        # Get the state of stores
+        pd-ctl -u http://<address>:<port> store | jq '.stores | .[] | .store.state_name'   
+        ```
+    
+    - [Access the TiDB cluster in Kubernetes](access-tidb.md) via MySQL client.
+
+## Step 3: Scale in the TiDB nodes of the source cluster
+
+Remove all TiDB nodes of the source cluster:
+
+- If the source cluster is deployed using TiUP, refer to [Scale in a TiDB cluster](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup#scale-in-a-tidbpdtikv-cluster).
+
+- If the source cluster is deployed using TiDB Ansible, refer to [Decrease the capacity of a TiDB node](https://docs.pingcap.com/tidb/stable/scale-tidb-using-ansible#decrease-the-capacity-of-a-tidb-node).
+
+> **Note:**
+>
+> If you access the source TiDB cluster via load balancing or database middleware, you need to first modify the configuration to route your application traffic to the target TiDB cluster. Otherwise, your application might be affected.
+
+## Step 4: Scale in the TiKV nodes of the source cluster
+
+Remove all TiKV nodes of the source cluster:
+
+- If the source cluster is deployed using TiUP, refer to [Scale in a TiKV cluster](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup#scale-in-a-tidbpdtikv-cluster).
+
+- If the source cluster is deployed using TiDB Ansible, refer to [Decrease the capacity of a TiKV node](https://docs.pingcap.com/tidb/stable/scale-tidb-using-ansible#decrease-the-capacity-of-a-tikv-node).
+
+> **Note:**
+>
+> * When you scale in the TiKV nodes one by one, wait until the store state of a TiKV node becomes "tombstone" and then scale in the next TiKV node.
+> * You can view the store state using PD Control.
+
+## Step 5: Scale in the PD nodes of the source cluster
+
+Remove all PD nodes of the source cluster:
+
+- If the source cluster is deployed using TiUP, refer to [Scale in a PD cluster](https://docs.pingcap.com/tidb/stable/scale-tidb-using-tiup#scale-in-a-tidbpdtikv-cluster).
+
+- If the source cluster is deployed using TiDB Ansible, refer to [Decrease the capacity of a PD node](https://docs.pingcap.com/tidb/stable/scale-tidb-using-ansible#decrease-the-capacity-of-a-pd-node).
+
+## Step 6: Delete the `spec.PDAddresses` field
+
+To avoid confusion for further operations on the cluster, it is recommended that you delete the `spec.PDAddresses` field in the new cluster after the migration. However, simply keeping the field does not have any impact on the cluster.
```

Comment:
```suggestion
To avoid confusion for further operations on the cluster, it is recommended that you delete the `spec.PDAddresses` field in the new cluster after the migration.
```

Reply:
Please also help remove the last sentence in the Chinese doc.